src/aiidalab_qe/plugins/pdos/setting.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""Panel for Pdos plugin."""
+import ipywidgets as ipw
+import traitlets as tl
+from aiida import orm
+from aiida_quantumespresso.calculations.functions.create_kpoints_from_distance import (
+    create_kpoints_from_distance,
+)
+
+from aiidalab_qe.common.panel import Panel
+
+# Periodicity
+NSCF_DISTANCE_MAP = {
+    "fast": 0.5,
+    "moderate": 0.1,
+    "precise": 0.05,
+}
+
+
+class Setting(Panel):
+    title = "Pdos Settings"
+    identifier = "pdos"
+    input_structure = tl.Instance(orm.StructureData, allow_none=True)
+    protocol = tl.Unicode(allow_none=True)
+
+    def __init__(self, **kwargs):
+        self.settings_title = ipw.HTML(
+            """<div style="padding-top: 0px; padding-bottom: 0px">
+            <h4>Settings</h4></div>"""
+        )
+        # nscf kpoints setting widget
+        self.nscf_kpoints_distance = ipw.BoundedFloatText(
+            min=0.0,
```

Comment:
```suggestion
            min=0.001,
```
and remove the `if else` in lines 61-69.